scripts/local/.env

```diff
@@ -15,8 +15,8 @@
 # PCS_IOTHUBREACT_ACCESS_CONNSTRING=HostName=iothub-test123.azure-devices.net;SharedAccessKeyName=iothubowner;SharedAccessKey=12345=
 # PCS_IOTHUB_CONNSTRING=HostName=iothub-test123.azure-devices.net;SharedAccessKeyName=iothubowner;SharedAccessKey=12345=
 # -----------------------------------------------------------------------------
-PCS_IOTHUBREACT_ACCESS_CONNSTRING
-PCS_IOTHUB_CONNSTRING
+PCS_IOTHUBREACT_ACCESS_CONNSTRING=HostName=iothub-o6j4v.azure-devices.net;SharedAccessKeyName=iothubowner;SharedAccessKey=TRLdINDFCXYZfIy/zMuAGjX2tRVfA9UkGayuqho7S0s=
```

Comment:
Remove the real connection string from the PR. Also please refresh the keys, this is a security issue. Never push the PR with connection strings.